### PR TITLE
Updated app list for raspbian

### PR DESCRIPTION
**Pull request by cntrlANC**

### CODE REVIEW

File: raspbian/applications/README.md [hunk 1]
```diff
@@ -4,3 +4,7 @@
     - Information on the demonstration camera software
 - [OMXPlayer](omxplayer.md)
     - Information on the OMXPlayer command line media player
+- [Minecraft for pi]
```
Comment (lurch) [inline raspbian/applications/README.md]: @JamesH65 Perhaps this is supposed to be linking to `usage/minecraft/README.md` ?

File: raspbian/applications/README.md [hunk 2]
```diff
@@ -4,3 +4,7 @@
...
+    - 3D block based building platform with its own simple 
+      line and command coding and also teaches electronic
+      circuitry.
```
Comment (lurch) [inline raspbian/applications/README.md]: @cntrlANC Just out of curiosity, how does Minecraft teach electronic circuitry?

File: raspbian/applications/README.md (hunk 1 above)
Comment (JamesH65) [reply raspbian/applications/README.md]: Have added link and removed the electronic circuit bit, which is a feature, but don't need to reference it here I don't think.

File: raspbian/applications/README.md (hunk 2 above)
Comment (XECDesign) [reply raspbian/applications/README.md]: Probably referring to redstone, but I don't think that works in the pi edition.

Comment (cntrlANC) [reply raspbian/applications/README.md]: yes.

Comment (cntrlANC) [reply raspbian/applications/README.md]: I can confirm that there is support for redstone as it is a more polished version. Minecraft pi edition also has support for extended programming languages.

**Comment by lurch** [comment]
Hmmm, on seconds thoughts, how do we determine what gets listed on https://github.com/raspberrypi/documentation/blob/master/raspbian/applications/README.md and what gets listed on https://github.com/raspberrypi/documentation/blob/master/usage/README.md ?
@JamesH65 @bennuttall 

_Perhaps_ it makes sense to move the pages under `raspbian/applications/` to instead be under `usage/` (with appropriate 'redirect pages' in place, to prevent external-links breaking) and then remove the "Applications" section from https://github.com/raspberrypi/documentation/blob/master/raspbian/README.md ?

**Comment by bennuttall** [comment]
> Hmmm, on second thought, how do we determine what gets listed on...

Just whatever makes sense

**Pull request merged**